labels-owners.yaml

```diff
@@ -0,0 +1,157 @@
+
+# This file provides information about labels and owners behind them
+#
+# Please keep this file alphabetized by label name and owner name
+
+repositories:
```

Comment:
Lets not group by repositories and kubeflow just remove those levels.

We should be applying these labels consistently.